docs/mastering-plone/features.md

```diff
@@ -274,7 +274,7 @@ Page
 
 News Item
 
-: Basically a page with an image and an image caption to be used for press releases an such.
+: Basically a page with an image and an image caption to be used for press releases as such.
```

Comment:
"and such" is the correct English phrase.

```suggestion
: Basically a page with an image and an image caption to be used for press releases and such.
```

Reply:
oh yes! Thanks!